client/src/pages/private/ParticipantTable.js

```diff
@@ -208,6 +208,7 @@ export default () => {
       const row = mapItemToColumns(item, columns);
 
       row.engage = item;
+      row.siteName = item.statusInfos && item.statusInfos.length > 0 && item.statusInfos[0].data?.siteName;
```

Comment:
Seem weird to have both short-circuiting and optional chaining.
```suggestion
      row.siteName = item?.statusInfos?.[0].data?.siteName;
```